com.unity.visualeffectgraph/Editor/FilterPopup/VFXFilterWindow.cs

```diff
@@ -589,13 +563,13 @@ private void ListGUI(Element[] tree, GroupElement parent)
 
             EditorGUIUtility.SetIconSize(new Vector2(16, 16));
 
-            List<Element> children = GetChildren(tree, parent);
+            Element[] children = GetChildren(tree, parent).ToArray();
```

Comment:
ToArray is known to be slightly faster than ToList. Here we don't need a list ...

Reply:
By the way, we aren't respecting all our coding rules. Here, it should have been `var` : https://ono.unity3d.com/unity-extra/unity-meta/files/@/ReferenceSource/CSharp/Assets/CSharpReference.cs#L222

Reply:
Thanks for the link I'll refer to it in the future when in doubt :)